client/components/banner/index.jsx

```diff
@@ -91,34 +104,40 @@ export class Banner extends Component {
 	}
 
 	handleClick = e => {
-		const { event, feature, onClick } = this.props;
+		const { event, feature, compact, onClick, tracksClickName, tracksClickProperties } = this.props;
 
-		if ( event ) {
-			this.props.recordTracksEvent( 'calypso_banner_cta_click', {
-				cta_name: event,
-				cta_feature: feature,
-				cta_size: 'regular',
-			} );
+		if ( ! tracksClickName ) {
+			return;
```

Comment:
I think we still want to call `onClick` when tracking is disabled. It might be also more secure to keep the `event` check so existing `Banner`s without defining the `event` prop ([example](https://github.com/Automattic/wp-calypso/blob/58188c18fee88c4c09bbdb31e2b85e75e7655fa8/client/blocks/privacy-policy-banner/index.jsx#L145-L152)) are not affected.

```js
if ( tracksClickName && event ) {
	this.props.recordTracksEvent( tracksClickName, {
		cta_name: event,
		cta_feature: feature,
		cta_size: compact ? 'compact' : 'regular',
		...tracksClickProperties,
	} );
}

onClick( e );
```

Reply:
Yikes good catch 😅 Let's not break existing instances. These should be fixed now!